recipes/protobuf/all/conanfile.py

```diff
@@ -109,6 +139,9 @@ def generate(self):
         if is_apple_os(self) and self.options.shared:
             # Workaround against SIP on macOS for consumers while invoking protoc when protobuf lib is shared
             tc.variables["CMAKE_INSTALL_RPATH"] = "@loader_path/../lib"
+        if Version(self.version) >= "4.22.0":
+            tc.variables["protobuf_ABSL_PROVIDER"] = "package"
+            tc.variables["CMAKE_CXX_STANDARD"] = 14
```

Comment:
it's possible to use self._min_cppstd without indentation and version checking

Reply:
fixed.

Reply:
Thanks!